website/app-templates/smarty/head.tpl

```diff
@@ -2,7 +2,15 @@
     <meta charset="utf-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <title>GeoKrety: {block name=title}{/block}</title>
-    <meta name="description" content="" />
+    <meta name="title" content="GeoKrety"/>
+    <meta name="description" content="Open source item tracking for all geocaching platforms"/>
```

Comment:
Do you think this string should be translatable?
If yes please surround them with `{t}{/t}`

Reply:
Yes 👍 that makes sense. I've added those tags.